### ISSUE TITLE
Retrieving Items (Shift "clicking") Extremely Slow/Inconsistent version: 1.9.4-2.1.0

### ISSUE THREAD

**Issue by Lanolin** [M4thG33k/TombManyGraves#2]
Hey just wanted to drop by and mention that shift "clicking" be it left, right, middle, left+right or whatever combination in between doesn't seem to reliably retrieve items from graves. Shift Right clicking will lock/unlock the grave, but retrieving items so far seems to just be me spamming some combination of shift "clicks" to EVENTUALLY get my items back. As I said I do EVENTUALLY get my items back, but it seems to be after a long delay, not so much after I shift "click" my grave.

TL;DR: Accessing the player's grave and thus retrieving your items seems extremely inconsistent. Shift "clicking" doesn't always return items in a timely manner.

version: 1.9.4-2.1.0

**Comment by M4thG33k** [comment]
You should obtain your items by sneaking and _walking_ over the grave. Clicking will only toggle the grave between locked/unlocked.

**Comment by Lanolin** [comment]
HA! You're right. I read and reread over the mod description countless times and I never noticed this! Thanks for the quick response.

**Comment by M4thG33k** [comment]
No problem! Thanks for playing with the mod!
